x-pack/legacy/plugins/siem/public/register_feature.ts

```diff
@@ -4,19 +4,18 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import {
-  FeatureCatalogueCategory,
-  FeatureCatalogueRegistryProvider,
-} from 'ui/registry/feature_catalogue';
+import { npSetup } from 'ui/new_platform';
+import { FeatureCatalogueCategory } from '../../../../../src/plugins/home/public';
 
 const APP_ID = 'siem';
 
-FeatureCatalogueRegistryProvider.register(() => ({
-  id: 'siem',
+// TODO(rylnd): move this into Plugin.setup once we're on NP
+npSetup.plugins.home.featureCatalogue.register({
+  id: APP_ID,
```

Comment:
FWIW we define `APP_ID` in `constants.ts` which is common to both server and client side and was at one point our single source of APP_ID truth.

You can use that one if you want and just import it:

```ts
x-pack/legacy/plugins/siem/common/constants.ts
```

optional, since you didn't add this one here and are just using the existing constant in this file which is also fine.

Reply:
Fixed!